File: atcoder/abc083/b2/abc083_b-2.go
package main

import "fmt"

func sumEachOrder(n int) int {
	sum := 0
	for f := n; f > 0; f /= 10 {
		sum += f % 10
	}
	return sum
}

func main() {
	var N, A, B int
	fmt.Scan(&N)
	fmt.Scan(&A)
	fmt.Scan(&B)

	ans := 0
	for n := 0; n <= N; n++ {
		s := sumEachOrder(n)
		if s >= A && s <= B {
			ans += n
		}
	}
	fmt.Println(ans)
}

File: atcoder/abc083/b2/abc083_b-2.py
N, A, B = [int(x) for x in input().split()]

def sumEachOrder(n):
    s = 0
#    print(n)
    while n > 0:
        a = n % 10
#        print("  {}".format(a))
        s += a
        n //= 10
    return s

ans=0
for n in range(1, N+1):
    #print(sumEachOrder(n))
    s = sumEachOrder(n)
    if s >= A and s <= B:
        ans += n
print(ans)
